lua/neogit/config.lua

```diff
@@ -135,6 +135,7 @@ end
 ---@field fetch_after_checkout? boolean Perform a fetch if the newly checked out branch has an upstream or pushRemote set
 ---@field telescope_sorter? function The sorter telescope will use
 ---@field disable_insert_on_commit? boolean|"auto" Disable automatically entering insert mode in commit dialogues
+---@field disable_editor_help? boolean|"auto" Disable displaying commit/rebase editor help
```

Comment:
I don't think I want to expose this - at least not now.

Reply:
Will remove this line.